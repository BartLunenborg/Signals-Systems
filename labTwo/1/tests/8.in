20: [39,42,42,21,-33,40,-47,-34,17,44,-2,-7,-15,33,-7,32,14,-5,-2,-49]
10: [-42,-8,-47,-22,-40,33,-13,37,35,18]
