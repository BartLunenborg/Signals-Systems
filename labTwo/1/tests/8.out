29: [-1638,-2076,-3933,-4050,-3240,-3720,1942,2093,5273,1412,6313,-2573,-676,-2583,17,-4274,765,-1204,789,-775,1753,3219,2344,3471,-710,640,-1973,-1751,-882]
